aws/resource_aws_rds_cluster_parameter_group_test.go

```diff
@@ -176,6 +176,25 @@ func TestAccAWSDBClusterParameterGroup_basic(t *testing.T) {
 					}),
 				),
 			},
+			{
+				Config: testAccAWSDBClusterParameterGroupUpdateParametersConfig(parameterGroupName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSDBClusterParameterGroupExists(resourceName, &v),
+					testAccCheckAWSDBClusterParameterGroupAttributes(&v, parameterGroupName),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.3127312348.name", "character_set_results"),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.3127312348.value", "ascii"),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.422535354.name", "character_set_server"),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.422535354.value", "ascii"),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.2478663599.name", "character_set_client"),
+					resource.TestCheckResourceAttr(
+						resourceName, "parameter.2478663599.value", "utf8"),
+				),
```

Comment:
these Set indices unfortunately are no longer supported in acceptance testing since the recent upgrade to V2 of the terraform-plugin-sdk, but they can be easily updated to use the new convention:
```go
tfawsresource.TestCheckTypeSetElemNestedAttrs(resourceName, "parameter.*", map[string]string{
    "name": "character_set_results",
    "value": "ascii"", 
}),
```
where the attribute that is a Set is provided with an `.*` suffix like `parameter.*` and the map[string]string provided corresponds to the key/value pairs expected